### PR TITLE
Export bug empty

### PR DESCRIPTION
**Pull request by naphets123**

### CODE REVIEW

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
[People Export Bug from IHA](https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha)

**Comment by naphets123** [comment]
https://trello.com/c/Kh2J2xpJ/436-people-export-bug-from-iha#

File: include/people_export.php [hunk 1]
```diff
@@ -5,7 +5,9 @@
 //Create a list of placeholders ? the same length as export ids given
 $id_pars = str_repeat('?,', count($export_ids_array) - 1).'?';
 //Put camp id as first element in the list
-array_unshift($export_ids_array, $_SESSION['camp']['id']);
+if ('' != $export_ids_array[0]) {
```
Comment (HaGuesto) [inline include/people_export.php]: @naphets123 If you do not add any value to `$export_ids_array`, does this not mean that an error is thrown two rows below when the SQL query is executed?

**Pull request merged**